packages/perspective-jupyterlab/test/config/jupyter/kernels/python3/kernel.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
???

Also the following two python icons - these are necessary additions to this PR?